get.php

```diff
@@ -24,7 +24,7 @@
 	"ccd-dev" => "https://github.com/nvdaes/clipContentsDesigner/releases/download/13.0/clipContentsDesigner-13.0.nvda-addon",
 	"ccd-o" => "https://github.com/nvdaes/clipContentsDesigner/releases/download/7.2/clipContentsDesigner-7.2.nvda-addon",
 	"ckbl" => "https://github.com/abdel792/sayCurrentKeyboardLanguage/releases/download/v19.12/sayCurrentKeyboardLanguage-19.12.nvda-addon",
-	"ckbl-dev" => "http://cyber25.free.fr/nvda-addons/sayCurrentKeyboardLanguage-19.12-dev.nvda-addon",
+	"ckbl-dev" => "https://github.com/abdel792/sayCurrentKeyboardLanguage/releases/download/v19.12/sayCurrentKeyboardLanguage-19.12.nvda-addon",
```

Comment:
This was previously a `-dev` version. The updated URL is not, I have downloaded both files and confirmed that the checksums do not match.

Since you are not GitHub user @abdel792 I'm not really sure what to do with this.